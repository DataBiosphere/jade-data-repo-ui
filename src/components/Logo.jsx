import React from 'react';
import styled from 'styled-components';

import Icon from 'assets/media/images/white-jade.png';

export const Logo = styled.img`
  align-items: flex-start;
  display: inline-flex;
  height: 41px;
  width: 41px;
`;

const Title = styled.span`
  color: #fff;
  position: relative;
  bottom: 12px;
  font-size: 21px;
  left: 4px;
`;

export default () => (
  <div>
    <Logo src={Icon} />
    <Title>Jade Data Repository</Title>
  </div>
);
