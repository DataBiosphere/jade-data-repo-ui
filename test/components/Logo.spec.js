import React from 'react';

import Logo from '../../src/components/Logo';

const ownProps = {
  classes: '',
};

function setup() {
  return mount(<Logo {...ownProps} />);
}

describe('Logo', () => {
  const wrapper = setup();

  it('should render correctly', () => {
    expect(wrapper).not.toBeNull();
  });
});
