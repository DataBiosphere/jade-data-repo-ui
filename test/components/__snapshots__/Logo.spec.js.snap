// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render properly 1`] = `
.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  height: 41px;
  width: 41px;
}

.c1 {
  color: #fff;
  position: relative;
  bottom: 12px;
  font-size: 21px;
  left: 4px;
}

<div>
  <Logo
    src={
      Object {
        "ReactComponent": "SvgMock",
      }
    }
  >
    <img
      className="c0"
      src={
        Object {
          "ReactComponent": "SvgMock",
        }
      }
    />
  </Logo>
  <Title>
    <span
      className="c1"
    >
      Jade Data Repository
    </span>
  </Title>
</div>
`;
